External/Plugins/ProjectManager/Controls/TreeView/OtherNodes.cs

```diff
@@ -33,7 +33,11 @@ public override void Refresh(bool recursive)
 
             if (References != null && References.Parent == null)
             {
-                if (recursive) RefreshReferences(recursive);
+                if (recursive)
+                {
+                    RefreshReferences(recursive);
```

Comment:
This causes an exception in `Nodes.Insert(0, References)`, because `RefreshReferences` already parents the references node.